src/Microsoft.TemplateEngine.Cli/TemplateInvocationCoordinator.cs

```diff
@@ -71,8 +73,9 @@ private async Task CheckForTemplateUpdateAsync(ITemplateMatchInfo templateToInvo
                 return;
             }
 
-            IReadOnlyList<CheckUpdateResult> versionChecks = await managedTemplateSource.ManagedProvider.GetLatestVersionsAsync(new[] { managedTemplateSource }, cancellationToken).ConfigureAwait(false);
-            if (versionChecks.Count == 1)
+            DefaultCredentialServiceUtility.SetupDefaultCredentialService(new CliNuGetLogger(), nonInteractive: true);
```

Comment:
What does it mean `nonInteractive: true`? That if "cached authentication" will work, but if it needs user input it will fail? How hard would it be to propagate `--interactive` flag down here?

Reply:
It's up to provider to define the behavior for `nonInteractive`. I've checked couple of providers in index - they fail if user prompt is needed but `nonInteractive` is true. 
We can read `--interactive` here - will fix that.